scripts/gen_docs.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e
+
+# Author: Thomas Bechtold <tbechtold@suse.com>
+# Author: Jeremy Moffitt <jmoffitt@suse.com>
+# License: Apache-2.0
+
+# List of project for running the doc generation
+#: ${PROJECTS:="aodh barbican ceilometer cinder designate glance heat horizon keystone magnum manila murano neutron neutron-fwaas neutron-lbaas neutron-vpnaas octavia nova sahara"}
+: ${PROJECTS:="keystone manila"}
```

Comment:
Yeah, e.g.

    source ~/.config/suse-openstack-docs

Reply:
good idea, moved these out to a .conf file, for the moment I put it in the scripts folder because I wanted to avoid creating another folder just for the one conf file after creating a scripts folder just for the scripts file... but I'm open to revisiting that... also realized that after splitting them out the script doesnt run correctly if its run from another directory since it looks for the conf file in the working folder... might need to revisit that